test/test_player.py

```diff
@@ -63,6 +89,305 @@ def test_backslash_in_arg(self):
         self.assertEqual(c.args, [u'hello \\ there'])
 
 
+class MPCResponse(object):
+    def __init__(self, raw_response):
+        self.body = b'\n'.join(raw_response.split(b'\n')[:-2])
+        self.status = raw_response.split(b'\n')[-2]
+        self.ok = (self.status.startswith(b'OK') or
+                   self.status.startswith(b'list_OK'))
+        self.err = self.status.startswith(b'ACK')
+        if not self.ok:
+            print(self.status.decode('utf-8'))
+
+
+class MPCClient(object):
+    def __init__(self, host, port, do_hello=True):
+        self.sock = socket.socket(socket.AF_INET, socket.SOCK_STREAM)
+        self.sock.connect((host, port))
+        self.buf = b''
+        if do_hello:
+            hello = self.get_response()
+            if not hello.ok:
+                raise RuntimeError('Bad hello: {}'.format(hello.status))
+
+    def __del__(self):
+        self.sock.close()
+
+    def get_response(self):
+        ''' Wait for a full server response and wrap it in a helper class.
+        If the request was a batch request then this will return a list of
+        `MPCResponse`s, one for each processed subcommand. '''
```

Comment:
In beets, we use [the PEP8 recommendations for docstrings](https://www.python.org/dev/peps/pep-0008/#documentation-strings), which include triple-double quotes, no spaces around the delimiters, and a newline before the closing delimiter:

```py
"""Like this.
"""
```